Vagrantfile

```diff
@@ -376,14 +373,14 @@ Vagrant.configure('2') do |config|
   end
 
   # Auto Download Vagrant plugins, supported from Vagrant 2.2.0
-  unless Vagrant.has_plugin?('vagrant-hostsupdater')
-    if File.file?(File.join(vagrant_dir, 'vagrant-hostsupdater.gem'))
-      system('vagrant plugin install ' + File.join(vagrant_dir, 'vagrant-hostsupdater.gem'))
-      File.delete(File.join(vagrant_dir, 'vagrant-hostsupdater.gem'))
-      puts "#{yellow}VVV has completed installing local plugins. Please run the requested command again.#{creset}"
+  unless Vagrant.has_plugin?('vagrant-hostmanager')
+    if File.file?(File.join(vagrant_dir, 'vagrant-hostmanager.gem'))
```

Comment:
This means that we need to update the cd usb generator

Reply:
Indeed, the instructions on the docs site and on .org will need adjusting too